docs/configuration.md

```diff
@@ -0,0 +1,186 @@
+```toml
+arch_subdir_discovery = "all"
+# or
+arch_subdir_discovery = { subdirs = ["linux-64", "osx-64", "win-64"] }
+```
+
+#### Repodata Patches
+  
+`repodata_patches_package`: Specifies a package that includes patches for "repodata". It is expected to be available in the channel.
+```toml
+repodata_patches_package = "conda-forge-repodata-patches"
+```
+
+#### Package Name to Feedstock Mapping
+
+`map_conda_forge_package_to_feedstock`: This is a flag used for enabling package name to feedstock name mapping for the `conda-forge` channel.
+If false, the package name is used as the feedstock name. If true, conda-forge's [feedstock-outputs](https://github.com/conda-forge/feedstock-outputs)
+is used to map the package name to the feedstock name.
+```toml
+map_conda_forge_package_to_feedstock = true
+```
+
+#### Provenance URL Pattern
+
+`provenance_url_pattern`: This provides a URL pattern to link to the source of a package. The `{feedstock}` placeholder is replaced with the feedstock name.
+```toml
+provenance_url_pattern = "https://github.com/conda-forge/{feedstock}-feedstock"
+```
+
+#### Package Filter
+
+`package_filter`: This property provides mechanism to filter packages by name or prefix.
+```toml
+# allows packages with names "r", "rpy2", "rstudio" and prefixes "r-", "_r-", "mro-"
+package_filter = { allowed_names = ["r", "rpy2", "rstudio"], allowed_prefixes = ["r-", "_r-", "mro-"] }
+```
+
+#### Broken Label Support
+
+`supports_broken_label`: This flag indicates that the channel supports a label called "broken" for releases that are not working (yanked).
+```toml
+supports_broken_label = true
+```
+
+#### Dashboards
+
+`dashboards`: List of dashboards that should be linked from the package browser. Dashboards are defined in the global `dashboards` section (see below).
+```toml
+dashboards = ["anaconda", "ghcr"]
+```
+
+#### Authentication
+
+`auth_username`, `auth_password`, `auth_quetz_token`, `auth_bearer_token`: Use these to configure authentication for the channel.
+
+```toml
+# HTTP Basic Authentication (for Artifactory)
+auth_username = "username"
+auth_password = "password"
+# or
+# Quetz Token Authentication
+auth_quetz_token = "token"
+# or
+# Bearer Token Authentication (for prefix.dev)
+auth_bearer_token = "pfx_token"
+```
+
+You can also configure those values to read from an environment variable or a secrets file. For example:
+```toml
+auth_username = { env = "ARTIFACTORY_USERNAME" }    # read from environment variable
+auth_password = { file = "/var/secrets/artifactory_password" }  # read password from file
+```
+
+#### Metadata Retrieval
+
+`metadata_retrieval`: This key indicates how metadata for a package will be retrieved. It supports two options:
+- `streamed`: Currently only supports .conda packages.
+- `oci_with_streamed_fallback`: Tries to use the OCI registry first, then falls back to streamed metadata.
+```toml
+metadata_retrieval = "streamed"
+```
+
+#### Override Extensions
```

Comment:
What is this needed for?

Reply:
The original app restricted the extensions for `pkgs/main` and `pkgs/r` to `.conda` and I simply copied that. Maybe you, @jaimergp, can explain this?

Original code:
https://github.com/Quansight-Labs/conda-metadata-app/blob/440cd247efd47ea504a26b16642102a61d2b9ee9/app.py#L206-L207

Reply:
Defaults is not mirrored to GHCR and we can't stream tar.bz2 files from anaconda.org, but defaults has .conda for all artifacts.